diff --git a/scripts/world/region2_ruins.gd b/scripts/world/region2_ruins.gd
index c942a22..6da2f24 100644
--- a/scripts/world/region2_ruins.gd
+++ b/scripts/world/region2_ruins.gd
@@ -20,6 +20,7 @@ static func move(state: Dictionary) -> Dictionary:
 		if state["room"] == link["from_room"] and state["cell"] == link["from_cell"]:
 			state["room"] = link["to_room"]
 			state["cell"] = link["to_cell"].duplicate()
+			if link["to_room"] == 1:state["cell"] = [22,5]
 			state["facing"] = link["facing"]
 			state["entry_lock"] = "region2_ruins_stairs"
 			break
